CHANGELOG.md

```diff
@@ -10,6 +10,12 @@
 
 #### Enhancements
 
+* Add new `redundant_extension` rule that detects redundant extensions.
+  An extension is considered redundant if it does not define any
+  members, but only conformances.
```

Comment:
The two spaces are required here to enforce a line break in the rendered Markdown:
```suggestion
  members, but only conformances.  
```